test/unittests/commands/xmlupload/make_rdf_graph/test_make_values.py

```diff
@@ -120,10 +116,9 @@ def test_geometry(self, lookups: IRILookups) -> None:
     def test_geoname(self, lookups: IRILookups) -> None:
         res_bn = BNode()
         prop = IntermediaryGeoname("5416656", absolute_iri("hasGeoname"), None, None)
-        result, prop_name = _make_one_value_graph(prop, res_bn, lookups)
+        result = _make_one_value_graph(prop, res_bn, lookups)
         assert len(result) == 3
-        assert prop_name == ONTO.hasGeoname
-        val_bn = next(result.objects(res_bn, prop_name))
+        val_bn = next(result.objects(res_bn, ONTO.hasGeoname))
```

Comment:
We do not need to assert the the property name, because if it is not the expected one, we would get a stop iteration here.